lib/livebook_web/live/output/grid_component.ex

```diff
@@ -48,4 +48,17 @@ defmodule LivebookWeb.Output.GridComponent do
     </div>
     """
   end
+
+  defp make_template(columns) when is_tuple(columns) do
+    columns = Tuple.to_list(columns)
+
+    if Enum.all?(columns, &is_integer/1) do
+      columns
+      |> Enum.map_join(" ", fn n -> "minmax(0, #{n}fr)" end)
+    else
+      ""
+    end
+  end
+
+  defp make_template(columns), do: "repeat(#{columns}, minmax(0, 1fr))"
```

Comment:
We want to make sure it is actually an integer. And we want a fallback in case we add more options in the future :)

```suggestion
  defp make_template(columns) when is_integer(columns), do: "repeat(#{columns}, minmax(0, 1fr))"

  defp make_template(_columns), do: ""
```